### ISSUE TITLE
Break I18n L10n code out into a stand alone module

### ISSUE THREAD

**Issue by ozten** [mozilla/persona#1322]
In order to reuse our I18n / L10n code in anothe rnode app, we should make a proper module out of it.

1) Create a functioning repo out of our code

https://github.com/mozilla/i18n-abide

2) Keep Mozilla specific code, config, and docs

3) Remove all other code and add i18n-abide as a dependency

Test plan - This should not affect or provide any new functionality.

**Comment by zaach** [comment]
Looks like we still need to do 2) and 3).

**Comment by ozten** [comment]
We're using i18n-abide on a few projects, it would be great to get this project using it... however it's probably a post-beta1 change (?).

**Comment by lloyd** [comment]
+1 - we should review all the changes I've made (gobbledeegook, etc) and verify we're ok with them, then update i18n abide.  also agree this is post-beta1

**Comment by shane-tomlinson** [comment]
@lloyd, @zaach, @ozten - This work has been done already it seems. Can this be closed?

**Comment by zaach** [comment]
We still need to switch over to the standalone module. The question is, should this be in the beta2 milestone?

**Comment by ozten** [comment]
`format` in browserid drifted away from `format` in i18n-abide.

See [example format usage of named variables](https://github.com/mozilla/browserid/blob/d494afbbce176697998f3b39764200d8732ad256/lib/i18n.js#L246) where in the format `%(foo)s`. Eventually, those could be typed... like %(foo)d for decimals, etc.

I'm going to switch these back, otherwise I have to break i18n-abide's format API. `%(foo)` doesn't seem to have any compelling reasons why we should switch the syntax.

**Comment by ozten** [comment]
Looks like this touches Gobbledygook.

https://github.com/lloyd/gobbledygook/pull/1

**Comment by ozten** [comment]
@mathjazz @shane-tomlinson I've fixed the strings in SVN r113372.

**NOTE:** the `.pot` files in `templates/LC_MESSAGES/` were not updated! Should those be?

I tried to do this in 1 step, but kept finding things I missed... so:

I noticed a few syntax errors and fixed those in r113375.

And then found lines that line-wrapped and didn't match my search and replace r113377.

And then I found signupBotton r113378.
